app/views/components/radios.scala.html

```diff
@@ -31,18 +31,9 @@
 )(implicit messages: Messages, userRequest: UserRequest[_], appConfig: AppConfig)
 
 @legend = {
```

Comment:
This isn't being used, according to the HMRC design system - there should be a `<h1>` inside the `<legend>` I suspect it is to do with the check on line 48.

Reply:
![image](https://user-images.githubusercontent.com/53828896/224283880-1bc50dff-48a8-492c-8712-a7513285cbc1.png)

Reply:
Be careful though as not all pages with fieldsets have legends which contain `<h1>` within them. For example the `/multiple-penalties-for-this-period` page has a h1 that isn't in the fieldset